verde/utils.py

```diff
@@ -14,6 +14,7 @@
 import pandas as pd
 import xarray as xr
 from scipy.spatial import cKDTree
+from sklearn.impute import KNNImputer
```

Comment:
Please use `verde.KNeighbors` instead.

Reply:
I used the KNeighbors class and used the predict and fit methods to fill in the values of the data array.